packages/flutter_tools/test/general.shard/flutter_platform_test.dart

```diff
@@ -15,6 +17,13 @@ import '../src/common.dart';
 import '../src/context.dart';
 
 void main() {
+  FileSystem fileSystem;
+
+  setUp(() {
+    fileSystem = MemoryFileSystem.test();
+    fileSystem.file('.packages').writeAsStringSync('\n');
```

Comment:
flutterPlatform.loadChannel reads the package config, which was hitting the real filesystem.